### ISSUE TITLE
Not working on Android

### ISSUE THREAD

**Issue by LRFalk01** [danielstern/ngAudio#16]
Running Chrome 37. I have ngAudio working on the desktop. On mobile I get nothing. I see the same result on your demo page. Any idea what is going on?

Edit: Looks like this is a Chrome feature . . .
https://code.google.com/p/chromium/issues/detail?id=178297

**Comment by Joenne** [comment]
I'm trying to use ngAudio in an Angular SPA. I want to automatically play some introductory sound in the first view. It works perfectly on a PC, but on an Android tablet (Chrome) I have to go to another view and back to the first view, to get the sound playing.
Is the only workaround forcing the user to click/touch something in the shell page? Clicking something in the view doesn't seem to work.

**Comment by danielstern** [comment]
Hey @Joenne 

It is this way on phones by design - so far I don't know of any way to get around this feature.

**Comment by Joenne** [comment]
Thank you!
I thought there was some kind of built in solution in ngAudio, since you earlier wrote "At long last this is fixed". I've now built my own fix on top of ngAudio, but it has the consequence that all audio elements must be placed at shell page level. I would prefer to place each audio element in the view, where it is going to be played.

**Comment by OriAlmogCode** [comment]
@Joenne do you mind sharing how you've done this? I'm trying to play music from a Nexus 5, so far playing one file works but switching files does not.

**Comment by Joenne** [comment]
@OriAlmogCode , I'll try:

1. I place all audio elements, that are going to be auto-played in the shell page. For convenience I give them a special class, viewaud (not given to other audio elements).
2. At startup I force the user to click a button. Normally I show a modal, that tells the user, that audio is an essential part of the application, offers a test of the speakers, and shows a continue-button.
3. In the continue-button's click-handler, I have something like this:

```
 angular.element('.viewaud').each(function(i,el){
               el.load();
                el.play();
                el.pause();
            })
```
4. I let my ngAudio elements refer to the audio elements in stead of referring directly to the sound files.
5. To avoid that ngAudio tries to start audio before the modal has finished, I set a scope variable to false at startup. Then I set it to true after the code above. Then I watch this variable and starts the audio, when it has changed to true.
